composer/core/state.py

```diff
@@ -73,9 +73,10 @@ class State(Serializable):
         precision (str | Precision): The numerical precision to use for training. See :class:`~.Precision` for
             the supported precisions.
         precision_context (Callable[[Precision], ContextManager]): Function to produce a context manager to mandate precision.
-        optimizers (:attr:`~.types.Optimizers`, optional): The optimizers being used to train the model. Multiple optimizers are not currently supported.
-        schedulers (:attr:`~.types.PyTorchScheduler` | List[:attr:`~.types.PyTorchScheduler`] | Tuple[:attr:`~.types.PyTorchScheduler`, ...], optional):
-            The learning rate scheduler (can also be a list or tuple of schedulers).
+        optimizers (:attr:`~.types.Optimizers`, optional): The optimizers being used to train the model.
```

Comment:
Following on to @jfrankle's feedback from #510, it would make the docstrings clearer if we did not use the plural types, and instead did something like:

```suggestion
        optimizers (types.Optimizer | Sequence[types.Optimizer], optional): The optimizers being used to train the model.
```
```suggestion
        optimizers (:attr:`~.types.Optimizers`, optional): The optimizers being used to train the model.
```